doc/tutorial/extending_theano.txt

```diff
@@ -0,0 +1,3 @@
+:orphan:
+This page has been moved. Please refer to: :ref:`extending_theano`. 
```

Comment:
Maybe he don't like the fact that there is no title?

Which version of sphinx do you have installed? Maybe if the version installed by travis is older, we could update it?

Maybe it the the first : from "to:" that he don't like? Or the orphan keyword?